app/services/cycle_timetable_query.rb

```diff
@@ -1,26 +1,5 @@
-class EndOfCycleTimetable
-  CURRENT_YEAR_FOR_SCHEDULE = 2021
-
-  # These dates are configuration for when the previous cycle ends and the next cycle starts
-  # The 2020 dates are made up so we can generate sensible test data
-  CYCLE_DATES = {
-    2020 => {
-      apply_1_deadline: Date.new(2019, 8, 24),
-      stop_applications_to_unavailable_course_options: Date.new(2019, 9, 7),
-      apply_2_deadline: Date.new(2019, 9, 18),
-      find_closes: Date.new(2019, 10, 3),
-      find_reopens: Date.new(2019, 10, 6),
-      apply_reopens: Date.new(2019, 10, 13),
-    },
-    2021 => {
-      apply_1_deadline: Date.new(2020, 8, 24),
-      stop_applications_to_unavailable_course_options: Date.new(2020, 9, 7),
-      apply_2_deadline: Date.new(2020, 9, 18),
-      find_closes: Date.new(2020, 10, 3),
-      find_reopens: Date.new(2020, 10, 6),
-      apply_reopens: Date.new(2020, 10, 13),
-    },
-  }.freeze
+class CycleTimetableQuery
+  RECRUITMENT_CYCLE_YEAR = 2021
```

Comment:
I don't like the fact this value is hard coded - could make this an env var? 
There is also a question about the relationship between this query object and the [RecruitmentCycle module](https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/models/recruitment_cycle.rb) - feels like there could be duplication of responsibility here?

Reply:
I think the progression is something like hardcoded > ENV var > automatically calculated from current date. But there's enough going on in this PR already I think?